libbeat/docs/gettingstarted.asciidoc

```diff
@@ -1,492 +1,17 @@
-you'll need to change the URL (IP:PORT) of the Elasticsearch server in the
-Kibana configuration file, `config/kibana.yml`, before starting Kibana.
-
-Use the following commands to download and run Kibana.
-
-*deb or rpm:*
-
-ifeval::["{release-state}"=="unreleased"]
-
-Version {stack-version} of Kibana has not yet been released.
-
-endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
-["source","sh",subs="attributes,callouts"]
-----------------------------------------------------------------------
-curl -L -O https://artifacts.elastic.co/downloads/kibana/kibana-{Kibana-version}-linux-x86_64.tar.gz
-tar xzvf kibana-{Kibana-version}-linux-x86_64.tar.gz
-cd kibana-{Kibana-version}-linux-x86_64/
-./bin/kibana
-----------------------------------------------------------------------
-
-endif::[]
-
-*mac:*
-
-ifeval::["{release-state}"=="unreleased"]
-
-Version {stack-version} of Kibana has not yet been released.
-
-endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
-["source","sh",subs="attributes,callouts"]
-----------------------------------------------------------------------
-curl -L -O https://artifacts.elastic.co/downloads/kibana/kibana-{Kibana-version}-darwin-x86_64.tar.gz
-tar xzvf kibana-{Kibana-version}-darwin-x86_64.tar.gz
-cd kibana-{Kibana-version}-darwin-x86_64/
-./bin/kibana
-----------------------------------------------------------------------
-
-endif::[]
-
-*win:*
-
-ifeval::["{release-state}"=="unreleased"]
-
-Version {stack-version} of Kibana has not yet been released.
-
-endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
-. Download the Kibana {Kibana-version} Windows zip file from the
-https://www.elastic.co/downloads/kibana[downloads page].
-
-. Extract the contents of the zip file to a directory on your computer, for example, `C:\Program Files`.
-
-. Open a command prompt as an Administrator and navigate to the directory that
-contains the extracted files, for example:
-+
-["source","sh",subs="attributes,callouts"]
-----------------------------------------------------------------------
-cd C:\Program Files\kibana-{Kibana-version}-windows
-----------------------------------------------------------------------
-
-. Run the following command to start Kibana:
-+
-["source","sh",subs="attributes,callouts"]
-----------------------------------------------------------------------
-bin\kibana.bat
-----------------------------------------------------------------------
-
-endif::[]
-
-You can find Kibana binaries for other operating systems on the
-https://www.elastic.co/downloads/kibana[Kibana downloads page].
-
-==== Launch the Kibana web interface
-
-To launch the Kibana web interface, point your browser to port 5601. For example,
-http://127.0.0.1:5601[http://127.0.0.1:5601].
-
-You can learn more about Kibana in the
-http://www.elastic.co/guide/en/kibana/current/index.html[Kibana User Guide].
+* {packetbeat-ref}/packetbeat-getting-started.html[Packetbeat]
```

Comment:
Should we alphabetize the list of Beats? The order seems arbitrary.

Reply:
Sort of. I ordered the list in a way that required the fewest changes to the diagram. :-) I don't have a strong preference either way because lists under 8 items are pretty easy for people to scan (we alphabetize mainly to make it easier for people to find stuff). Since you brought it up, though, I will alphabetize the list.